.travis.yml

```diff
@@ -4,13 +4,15 @@ language: ruby
 cache: bundler
 env:
 - INTEGRATION=false
-- INTEGRATION=true ES_VERSION=1.7.6 TEST_DEBUG=true
-- INTEGRATION=true ES_VERSION=2.4.4 TEST_DEBUG=true
-- INTEGRATION=true ES_VERSION=5.6.9 TEST_DEBUG=true
-- INTEGRATION=true ES_VERSION=6.2.4 TEST_DEBUG=true
-- INTEGRATION=true ES_VERSION=master TEST_DEBUG=true
-- SECURE_INTEGRATION=true INTEGRATION=true ES_VERSION=5.6.9 TEST_DEBUG=true
-- SECURE_INTEGRATION=true INTEGRATION=true ES_VERSION=6.2.4 TEST_DEBUG=true
+- DISTRIBUTION=legacy INTEGRATION=true ES_VERSION=1.7.6 TEST_DEBUG=true
+- DISTRIBUTION=legacy INTEGRATION=true ES_VERSION=2.4.4 TEST_DEBUG=true
+- DISTRIBUTION=legacy INTEGRATION=true ES_VERSION=5.6.9 TEST_DEBUG=true
+- DISTRIBUTION=oss INTEGRATION=true ES_VERSION=6.4.2 TEST_DEBUG=true
```

Comment:
If we removed oss testing (only testing the default package) then we could likely get rid of the whole DISTRIBUTION environment variable, since 
- legacy means < 6.3.0
- legacyxpack means < 6.3.0 + xpack (which we know we need because of SECURE_INTEGRATION=true)
- default >= 6.3.0

Reply:
I wanted to have the `OSS` testing for ILM when it comes - as it is an xpack feature, there will be different tests for open source and xpack flavours of elasticsearch

Reply:
testing the oss won't give us more information as it's a subset of the codebase of the default distro

Reply:
oops, I only now read your comment, got it, makes sense.